tests/test_configurations/test_loader.py

```diff
@@ -101,27 +101,37 @@ def test_config_loader_load_service_config(*_mocks):
             "agent": "agent",
             "network": "hardhat",
             "number_of_agents": "4",
+            "overrides": [],
         }
     )
 
+    class DummyServiceConfig(PackageConfiguration):
+
+        package_type = PackageType.SERVICE
+        json = property(lambda self: config)
+
+        def pop(self, item):
+            return self.json.pop(item)
+
+    dummy_service_config = DummyServiceConfig(name="Service", author="valory")
+
     with TemporaryDirectory() as temp_dir:
         schema_file = Path(temp_dir, "schema.json").absolute()
         config_file = Path(temp_dir, "service.yaml")
         schema_file.write_text("{}")
 
-        config_loader_cls = MagicMock()
+        config_loader_cls = DummyServiceConfig
         config_loader_cls.schema = str(schema_file)
-        config_loader_cls.from_json = lambda x: MagicMock(**x)
-
+        config_loader_cls.from_json = lambda x: dummy_service_config
         config_loader = ConfigLoader.from_configuration_type(
             PackageType.SERVICE, {PackageType.SERVICE: config_loader_cls}
         )
 
-        dummy_obj = MagicMock()
-        dummy_obj.package_type = PackageType.SERVICE
-        dummy_obj.ordered_json = config
+        config_file.touch()
+        with pytest.raises(ValueError, match="Agent configuration file was empty."):
```

Comment:
What `Agent configuration file was empty.` error message has to do with the service config test

Reply:
well absolutely nothing of course. Some deviant character has copy pasted code it seems!

https://github.com/valory-xyz/open-aea/blob/5a78c4cebe8d61a7321df01fbd1cfe47397a1709/aea/configurations/loader.py#L273-L274
https://github.com/valory-xyz/open-aea/blob/5a78c4cebe8d61a7321df01fbd1cfe47397a1709/aea/configurations/loader.py#L321-L325

Reply:
Please update the error message